pkg/seeds/seeds.go

```diff
@@ -16,19 +17,16 @@ type SeedOptions struct {
 func SeedRepositoryConfigurations(db *gorm.DB, size int, options SeedOptions) error {
 	var repos []models.RepositoryConfiguration
 
-	if options.OrgID == "" {
-		options.OrgID = fmt.Sprintf("%d", rand.Intn(9999))
-	}
-
 	for i := 0; i < size; i++ {
 		repoConfig := models.RepositoryConfiguration{
 			Name:      fmt.Sprintf("%s - %s - %s", RandStringBytes(2), "TestRepo", RandStringBytes(10)),
 			URL:       fmt.Sprintf("https://%s.com/%s", RandStringBytes(20), RandStringBytes(5)),
-			Versions:  []string{"9"},
-			Arch:      "x86_64",
-			AccountID: fmt.Sprintf("%d", rand.Intn(9999)),
-			OrgID:     options.OrgID,
+			Versions:  createVersionArray(i),
+			Arch:      createArch(i),
+			AccountID: strconv.Itoa(rand.Intn(9999)),
+			OrgID:     createOrgId(options.OrgID, i),
```

Comment:
If I'm understanding this right, you're using the index of the for loop to help create random values. Would it be easier to create random values without depending on the for loop and keep the randomness entirely within the function?

Reply:
This was intentional, I want predictable randomness for tests. Without the addition of the index, the outcomes of repo counts on particular filters were variable. Now they are not.

Reply:
Would that predictable randomness be, for example, if you wanted your seeded data to have 5 repos with version 9? Something like that might be better as some kind of seedOptions variable, but I'm not sure what that would look like.

Also, I believe the randomness here depends on how many repos you seed at a time, which feels odd. Maybe I just want to seed a few repos for local testing.

But we don't have to block the PR on this. I'm assuming it works okay if the tests are passing!

Reply:
It does work as today, most tests use a sample size of 20 in the DB, and this seed allows for at least 1 of each type filter within that small of sample.